README.md

```diff
@@ -1 +1,6 @@
-# advent-of-code-2015
\ No newline at end of file
+# Advent of Code 2015
+
+My attempt at https://adventofcode.com/2015 in Common Lisp.
+
+## Running
+TBD
```

Comment:
I wouldn't mind this, even if its just for posterity

Reply:
Done.